qwik/lib/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "my-qwik-library-name",
+  "version": "0.0.1",
+  "description": "Create a Qwik library",
+  "main": "./lib/index.qwik.mjs",
+  "qwik": "./lib/index.qwik.mjs",
+  "types": "./lib-types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/index.qwik.mjs",
+      "require": "./lib/index.qwik.cjs",
+      "types": "./lib-types/index.d.ts"
+    }
+  },
+  "files": [
+    "lib",
+    "lib-types"
+  ],
+  "engines": {
+    "node": "^18.17.0 || ^20.3.0 || >=21.0.0"
+  },
+  "private": false,
+  "type": "module",
+  "scripts": {
+    "build": "qwik build",
```

Comment:
we should swap these with Bazel commands that do the equivalent thing

Reply:
Added the build, test and start bazel commands to the package.json files. I left most of the other scripts as to not get too far from the examples. Do say if you wish to leave just the bazel build stuff.

Reply:
perfect